src/Magick.NET/Helpers/ByteArrayWrapper.cs

```diff
@@ -3,34 +3,76 @@
 
 using System;
 using System.IO;
+#if NETSTANDARD2_1_OR_GREATER
+using System.Buffers;
+#endif
 
 namespace ImageMagick;
 
 internal sealed unsafe class ByteArrayWrapper
 {
+    #if NETSTANDARD2_1_OR_GREATER
+
+    private static readonly ArrayPool<byte> _pool = ArrayPool<byte>.Create(1024 * 1024 * 64, 1024);
```

Comment:
I wonder if we should move this logic to a separate class so we can also use this in other places but I will do that at a later moment myself.